api/Window.json

```
@@ -2011,20 +2011,38 @@
               "partial_implementation": true,
               "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
             },
-            "firefox": {
-              "version_added": "29"
-            },
-            "firefox_android": {
-              "version_added": "32"
-            },
+            "firefox": [
+              {
+                "version_added": "89"
+              },
+              {
+                "version_added": "29",
+                "version_removed": "89",
+                "partial_implementation": true,
+                "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://bugzil.la/1697093'>bug 1697093</a>."
```

Suggesting some changes to make this consistent with https://github.com/mdn/browser-compat-data/pull/17061:

```suggestion
                "notes": "The <code>ongamepadconnected</code> event handler property is not supported. See <a href='https://bugzil.la/1697093'>bug 1697093</a>."
```